### PR TITLE
[202012][multi-asic][sonic-config-engine]: Get PORT table from namespace config db

### PR DESCRIPTION
**Pull request by SuvarnaMeenakshi**
Why I did it
Cherry-pick of: https://github.com/Azure/sonic-buildimage/pull/7632
portconfig.py gets PORT table from config_db if it is present. If not, port_config.ini files are parsed.
For multi-asic platform, if namespace is passed to get_port_config(), config_db connection was done to host namespace always and not to asic specific namespace.
Provides fix for: https://github.com/Azure/sonic-buildimage/issues/7161

How I did it
Modify db connection function to connect to namespace config_db.

#### How to verify it
Unit-test passed.
Verified on multi-asic VS platform.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

**Review by wenyiz2021 — APPROVED** [review]

File: src/sonic-config-engine/portconfig.py [hunk 1]
```diff
@@ -46,11 +59,16 @@ def readJson(filename):
...
+def db_connect_configdb(namespace=None):
     """
     Connect to configdb
     """
-    config_db = ConfigDBConnector()
+    try:
+        if namespace is not None:
+            swsscommon.SonicDBConfig.load_sonic_global_db_config(namespace=namespace)
+        config_db = swsscommon.ConfigDBConnector(use_unix_socket_path=True, namespace=namespace)
+    except Exception as e:
```
Comment (qiluo-msft) [inline src/sonic-config-engine/portconfig.py:70]: >Exception [](http://example.com/codeflow?start=11&length=9)

What is the type of exception you are expecting? The general exception type will hide bugs and should be limited.

Comment (SuvarnaMeenakshi) [reply src/sonic-config-engine/portconfig.py:70]: Was added to handle exception if thrown by load_sonic_global_db_config().

Comment (wenyiz2021) [reply src/sonic-config-engine/portconfig.py:70]: looks like load_sonic_global_db_config() calls SonicDBConfig::initializeGlobalConfig() which already handled exceptions, and it should be good enough?

Comment (SuvarnaMeenakshi) [reply src/sonic-config-engine/portconfig.py:70]: I have created a PR : https://github.com/Azure/sonic-buildimage/pull/10581 to remove this, I am working on unit-test failure, will merge that PR in master branch first and then take it in to 202012

Comment (SuvarnaMeenakshi) [reply src/sonic-config-engine/portconfig.py:70]: @qiluo-msft https://github.com/Azure/sonic-buildimage/pull/10581 is merged in master branch to fix this comment. Can we merge this PR to 202012 branch and cherry-pick the other PR to 202012.

**Review by qiluo-msft — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

File: src/sonic-config-engine/portconfig.py (hunk 1 above)
Comment (SuvarnaMeenakshi) [reply src/sonic-config-engine/portconfig.py:70]: https://github.com/Azure/sonic-buildimage/pull/10960 - PR raised to fix this comment.

**Pull request merged**